apps/calendar/build/calendar.build.js

```diff
@@ -17,15 +18,34 @@
 
   modules: [
     {
-      name: 'ext/alameda',
+      create: true,
```

Comment:
before it was just compiling the `ext/alameda` and `include`'ing the other modules.. now it creates a new file called `bundle.js` which makes way more sense than having a bunch of stuff that is not part of `alameda` inside the `ext/alameda.js` file